actions/actions.py

```diff
@@ -264,6 +264,23 @@ def run(self, dispatcher: CollectingDispatcher,
         return[SlotSet("contact", name), SlotSet("title", None)]
 
 
+class ActionGetOperator(Action):
+    """Here is where some kind of human connection mechanism would be implemented."""
+    def name(self) -> Text:
+        return "action_get_operator"
+
+    def run(
+        self,
+        dispatcher: "CollectingDispatcher",
+        tracker: Tracker,
+        domain: "DomainDict",
+    ) -> List[Dict[Text, Any]]:
+
+        dispatcher.utter_message(response="utter_operator")
+
+        return []
+
+
```

Comment:
So on our gitlab side, this placeholder action at least queries the KB for the main office phone and email - should I add that function to this repo, along with the Andabær office info to the RDF?

Reply:
yes